examples/spot/lightning_cifar10/requirements.txt

```diff
@@ -1,6 +1,6 @@
 pytorch-lightning>=1.3
 torchvision
 wandb
-torchmetrics>=0.3
+torchmetrics==0.4.1
```

Comment:
How about >= 0.4.1 so future updates can be supported?

Reply:
The issue is, having the recent version, 0.11.4,downloaded what caused issue. So I fixed rather with minimal working version.